aim/cli/up/utils.py

```diff
@@ -27,3 +32,112 @@ def build_uvicorn_command(host, port, num_workers, ssl_keyfile, ssl_certfile):
         cmd += ['--ssl-certfile', ssl_certfile]
     cmd += ['aim.web.run:app']
     return cmd
+
+
+def parse_tf_events(tf_logs, repo_inst):
+    """
+    This function scans and collects records from TF event files.
+
+    Creates and uses cache file "tf_logs_cache.json" in the repo dir
+    to track previously processed files and values
+
+    Currently, supports the following TF plugins
+        - scalar
+        - image
+    """
+
+    # This import statement takes long to complete
+    if not os.path.isdir(tf_logs):
+        logger.error('Value of "tf_logs" should be a valid path to the root of TF logdir.')
+        return
+
+    try:
+        import tensorflow as tf
+        from tensorflow.python.summary.summary_iterator import summary_iterator
+    except ModuleNotFoundError:
+        logger.error("Cannot process TF events - could not import required modules.")
+        return
+
+    tf_logs_cache_path = f'{repo_inst.path}/tf_logs_cache.json'
+    try:
+        with open(tf_logs_cache_path) as FS:
+            tf_logs_cache = json.load(FS)
+    except Exception:
+        tf_logs_cache = {}
+
+    for run_path in os.listdir(tf_logs):
+        path = os.path.join(tf_logs, run_path)
+        if path not in tf_logs_cache:
+            tf_logs_cache[path] = {}
+
+        run_cache = tf_logs_cache[path]
+        if run_cache:
+            run = Run(repo=repo_inst, run_hash=run_cache['run_hash'])
```

Comment:
need to pass `system_tracking_interval=None` to disable resource usage tracking.